storage/client/backend.go

```diff
@@ -5,6 +5,7 @@ import (
 	"log"
 
 	pb "github.com/opiproject/opi-api/storage/v1/gen/go"
+	pc "github.com/opiproject/opi-api/common/v1/gen/go"
```

Comment:
@PWAlessi is that how that is actually done in golang ?

Reply:
You can use any name to alias it. I think that `common` or something like that would be more intuitive